packages/block-editor/src/components/text-decoration-control/index.js

```diff
@@ -2,10 +2,19 @@
  * WordPress dependencies
  */
 import { Button } from '@wordpress/components';
-import { formatStrikethrough, formatUnderline } from '@wordpress/icons';
+import {
+	formatStrikethrough,
+	formatUnderline,
+	textColor,
+} from '@wordpress/icons';
 import { __ } from '@wordpress/i18n';
 
 const TEXT_DECORATIONS = [
+	{
+		name: __( 'None' ),
+		value: 'none',
+		icon: textColor,
```

Comment:
I could use some suggestions about what icon to use here for indicating `text-decoration: none` cc @jasmussen @jameskoster

Reply:
Thanks for the ping. 

> Adds a "None" option to text decoration controls, for overriding the theme settings.

The case for "overriding theme settings" here is the common case of someting like `a { text-decoration: underline; }` correct? Or were you referring to some theme.json provided switch?

In any case, from what I can tell there already exists a "none" button:

<img width="591" alt="Screenshot 2021-05-10 at 08 08 46" src="https://user-images.githubusercontent.com/1204802/117613560-60e06a80-b167-11eb-9b3f-caa6ecb57e32.png">

<img width="106" alt="Screenshot 2021-05-10 at 08 08 28" src="https://user-images.githubusercontent.com/1204802/117613565-62119780-b167-11eb-8a83-51483f416c64.png">

The icon is not the great, and while "underline" is fine, the three of them together aren't harmoniously balanced to sit next to each other. I like what figma does here, having a generic "none" icon that is reused for every property that has a "none":

<img width="249" alt="Screenshot 2021-05-10 at 08 09 30" src="https://user-images.githubusercontent.com/1204802/117613650-7fdefc80-b167-11eb-90db-2d069e401a02.png">

However updated icons feels like a task for another day, and shouldn't block this effort.

Reply:
If this bit is extracted to a different PR it can land faster (I'm not sure about the state of the other things I've mentioned at https://github.com/WordPress/gutenberg/pull/31623/files#r629277217 may or may not be ready to land but we need them for this).